docs/rm-eq.rst

```diff
@@ -70,7 +74,12 @@ An example JSON claim
     "iat":"1458047712",
     "exp":"1458057712",
     "tx_id": "0f534ffc-9442-414c-b39f-a756b4adc6cb",
-    "region_code": "GB-GBN"
+    "region_code": "GB-GBN",
+    "language_code": "en",
+    "variant_flags": {
+      "flag_1": true,
+      "flag_2": "something"
+    }
```

Comment:
I think we should restrict the variant flag values to booleans. We only have one use case for this at the moment and it would make sense to keep this as simple as possible.